python/cuspatial/cuspatial/_lib/spatial_window.pyx

```diff
@@ -0,0 +1,33 @@
+# Copyright (c) 2019, NVIDIA CORPORATION.
+
+from cudf._lib.table cimport table, Table
+from cudf._lib.move cimport move, unique_ptr
+from cudf._lib.column cimport column, Column
+
+cpdef points_in_spatial_window(
```

Comment:
I think this can be cdef. the rules behind def, cdef, and cpdef are weird. I'm not sure if any one of them effects compilation time, but it takes practically no time to compile cython compared to the rest of the code base anyway, so my comment is mostly irrelevant.

Reply:
When I change this to cdef the python function fails to call it, and I get a Cython warning about unused function. I think I'll leave it.